apps/parsley/cypress/support/index.ts

```diff
@@ -114,7 +114,9 @@ declare global {
     cy.login();
     cy.setCookie("has-opened-drawer", "true");
     cy.setCookie("has-seen-searchbar-guide-cue", "true");
-    cy.setCookie("has-seen-jump-to-failing-line-guide-cue", "true");
+    cy.setCookie("has-seen-sections-beta-guide-cue", "true");
+    cy.setCookie("seen-sections-beta-feature-modal", "true");
```

Comment:
This isn't used in the codebase. 
Also these are declared both here and in the beforeEach hook for featureDiscovery.ts are they both necessary?

Reply:
You can see the usage for these cookies by searching `SEEN_SECTIONS_BETA_GUIDE_CUE` and `SEEN_SECTIONS_BETA_FEATURE_MODAL`. By setting them to "true", the intro guide cues and modals will be hidden by default for all tests. They are set to "false" in `featureDiscovery.ts` so they can be tested.